lib_eio/unix/fork/dune

```diff
@@ -0,0 +1,4 @@
+(library

```

Comment:
Does this need to be split out, or just a left-over change?

Reply:
Left-over before the ifdefs thanks!